Makefile

```diff
@@ -145,10 +145,11 @@ HADOLINT_FAILURE_THRESHOLD = warning
 
 SHELLCHECK_VER := v0.9.0
 
+# KPROMO_PKG may have to be changed if KPROMO_VER increases its major version.
+KPROMO_PKG := sigs.k8s.io/promo-tools/v4/cmd/kpromo
 KPROMO_VER := v4.0.4
 KPROMO_BIN := kpromo
 KPROMO :=  $(abspath $(TOOLS_BIN_DIR)/$(KPROMO_BIN)-$(KPROMO_VER))
```

Comment:
Can we move l.148+149 down again? We're just trying to keep this sync in some other repos and it's annoying if we move it around :D

Reply:
Good point

Reply:
Sure thing, done!